client/src/components/PageLayout/Header.tsx

```diff
@@ -53,75 +52,74 @@ export const Header: React.FC = () => {
           />
         </Link>
         <HStack as="nav">
-          <Box>
-            {loadingUser ? (
-              <Spinner color="white" size="xl" />
-            ) : (
-              <Menu>
-                <MenuButton
-                  as={Button}
-                  data-cy="menu-button"
-                  aria-label="Options"
-                  variant="outline"
-                  background="gray.10"
-                  px={[2, 4]}
-                  py={[1, 2]}
-                >
-                  Menu
-                </MenuButton>
-                <MenuList paddingBlock={0}>
-                  <Flex
-                    flexDirection="column"
-                    fontWeight="600"
-                    borderRadius="5px"
-                  >
-                    <NextLink passHref href="/chapters">
-                      <MenuItem as="a">Chapters</MenuItem>
-                    </NextLink>
+          {loadingUser ? (
+            <Spinner color="white" size="xl" />
+          ) : (
+            <Menu>
+              <MenuButton
+                as={Button}
+                data-cy="menu-button"
+                aria-label="Options"
+                variant="outline"
+                background="gray.10"
+                px={[2, 4]}
+                py={[1, 2]}
+              >
+                Menu
+              </MenuButton>
+              <MenuList
+                paddingBlock={0}
+                display="flex"
+                flexDirection="column"
+                fontWeight="600"
+                borderRadius="5px"
+              >
+                {/* We are using Nextlink because of confustion in NextJs which harm the loading functionality and force the page to wait for the JS loading to display it */}
```

Comment:
I don't really understand this, could you explain?  The purpose of next/link is to handle within app page transitions, which it can only do once the js has been evaluated (but it's still just an anchor element until then).

Reply:
Since it's talking about behaviour we're not running into now, yeah, let's get rid of it.